test/__snapshots__/StatsTestCases.basictest.js.snap

```diff
@@ -3395,8 +3395,8 @@ cacheable modules X bytes
   ./first.js X bytes [built] [code generated]
   ./second.js X bytes [built] [code generated]
   ./vendor.js X bytes [built] [code generated]
-  ./module_first.js X bytes [built] [code generated]
   ./common2.js X bytes [built] [code generated]
+  ./module_first.js X bytes [built] [code generated]
```

Comment:
Can we verify is it a right change?

Reply:
I don't understand this change (it's changing the order)